src/main/java/com/redhat/cloud/notifications/db/BundleResources.java

```diff
@@ -38,6 +38,13 @@
         return session.find(Bundle.class, id);
     }
 
+    public Uni<Bundle> getBundle(String name) {
+        String query = "FROM Bundle WHERE name = :name";
+        return session.createQuery(query, Bundle.class)
+                .setParameter("name", name)
+                .getSingleResult();
```

Comment:
We probably don't want a `NoResultException` to be thrown if the given name is unknown:
```suggestion
                .getSingleResultOrNull();
```
Since this didn't show up in tests, we certainly have one uncovered case (user config retrieval with an uknown bundle name).

Reply:
I think I added a test with that, will double check.

Reply:
I think there is no test for this case. I'll add one (I'm OK with failing the request)